js/npm/etherscan/README.md

```diff
@@ -0,0 +1,34 @@
+# @parity/Etherscan
+
+A thin, lightweight promise wrapper for the api.etherscan.io/apis service, exposing a common endpoint for use in JavaScript applications.
+
+[https://github.com/ethcore/parity/js/src/3rdparty/etherscan](https://github.com/ethcore/parity/js/src/3rdparty/etherscan)
```

Comment:
Updated.

As for the README, do agree, but actually need to solve this with the source in npm issue as well. Will log and revisit.